main/boofcv-ip/src/test/java/boofcv/alg/misc/impl/TestImplImageBandMath_MT.java

```diff
@@ -0,0 +1,301 @@
+package boofcv.alg.misc.impl;
```

Comment:
Similar comment here as in TestImageBandMath. This test should compare it against ImplImageBandMath. The tests will almost be identical, but change the width and height to be 80 and 90 so that the concurrent code has more of a chance to process the image in non-sequential order.

Reply:
I set the size to 80x90, but otherwise do not understand the comment.  Do you also want to use reflection here to check the functions?  It takes me too much time to work myself into what exactly is going in the reflection based test code, and this seems to do what we want from it (but I may be missing something).